doc/Programming-guide/Key-platform-features/Endpoint-profiles/index.md

```diff
@@ -300,20 +309,19 @@ profile->destroy(profile);
 </div>
 </div>
 
-## Server-side endpoint profile ##
+## Server-side endpoint profile
 
-The server-side endpoint profile is initially generated at the stage of a new endpoint registration. By default, server side profile record is auto-generated based on 
-the latest server-side profile schema of particular application. 
+The server-side endpoint profile is initially generated at the stage of a new endpoint registration.
+By default, server-side profile record is auto-generated based on the latest server-side profile schema of a particular application.
 In this case, endpoint membership in the endpoint groups is re-evaluated and updated to match the new endpoint profile.
-For creating server side endpoint profile by REST API first of all you need to 
-[create new CT schema]({{root_url}}Programming-guide/Server-REST-APIs/#!/Common_Type_Library/saveCTLSchemaWithAppToken)
-after that [create server-side profile schema]({{root_url}}Programming-guide/Server-REST-APIs/#!/Profiling/saveServerProfileSchema) which will be contain a reference for this CT.
+To create a server-side endpoint profile using a REST API, you need to [create new CT]({{root_url}}Programming-guide/Server-REST-APIs/#!/Common_Type_Library/saveCTLSchemaWithAppToken).
```

Comment:
Moreover, you don't necessarily **have to** create a new CT. You can choose among the existing ones.